### DISCUSSION TITLE
"Pathway" mod suggestion

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#20100] [Ideas]
Some kind of "follow" mod:
- Follow points act as a pathway.
  - If you go out of this path with your cursor you will be penalized. Ideas:
    - Miss / fail
    - A big HP loss
    - One of the previous penalties but after X seconds / over time
    - Make it count as invalid for submission
  - You can see the path (Opacity customisation? Hidden with some specific mods like `Flashlight` or `Hidden`?)
    Probably NOT as shown below (an opaque white line?). It's only for illustration.
- The path size could be based on a beatmap attribute:
  - Accuracy
  - Star Rating
  - Circle Size
  - others (not sure)

Might count as a restrictive mod, but looking at the requirements (https://github.com/ppy/osu/discussions/20056), I'm not sure.

![image](https://user-images.githubusercontent.com/43073074/188130610-81ad312f-3b5d-4880-ab97-e10588c7dc1e.png)

**Comment by Gabixel** [comment]
I don't have much idea how playable it is, at first glance, when I thought of it it seemed valid, but it should be discussed for sure

**Reply by Doksha** [reply]
This seems to me like more of just a "dont have fun" mod, it doesnt make normal gameplay harder (unless I am misunderstanding the idea) it just prevents you from cursor dancing, and there is already the strict tracking mod for sliders.

**Reply by Walavouchey** [reply]
and continuing along that train of thought, maybe this kind of mode should be a setting on the strict tracking mod. the name makes sense at least
